### ISSUE TITLE
Yet Another Error

### ISSUE THREAD

**Issue by megahoy** [safarijv/yet-another-django-profiler#20]
```
Traceback (most recent call last):
  File "/usr/local/bin/gprof2dot.py", line 3184, in <module>
    main()
  File "/usr/local/bin/gprof2dot.py", line 3145, in main
    profile = parser.parse()
  File "/usr/local/bin/gprof2dot.py", line 2592, in parse
    callee = self.get_function(fn)
  File "/usr/local/bin/gprof2dot.py", line 2580, in get_function
    name = self.get_function_name(key)
  File "/usr/local/bin/gprof2dot.py", line 2572, in get_function_name
    module = settings.path_to_module_function(filename)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/functional.py", line 33, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python2.7/dist-packages/yet_another_django_profiler/conf.py", line 103, in path_to_module_function
    mod_name, function_name = self.YADP_PATH_TO_MODULE_FUNCTION.rsplit('.', 1)
  File "/usr/local/lib/python2.7/dist-packages/yet_another_django_profiler/conf.py", line 72, in YADP_PATH_TO_MODULE_FUNCTION
    'yet_another_django_profiler.conf.path_to_module_name')
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 55, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 43, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python2.7/dist-packages/django/conf/__init__.py", line 99, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named ayk.settings
```

It fails when I am using http://domain/?profile, but with http://domain/?profile=time it's working fine.

P.S. `ayk` is my application name

**Comment by jieter** [comment]
Same here...

**Comment by leehinde** [comment]
Me too. (sorry) but I'll add that I have graphiz installed via homebrew on El Capitan.

**Comment by jmbowman** [comment]
This looks like a duplicate of #16, which I just fixed in version 1.0.3.  Could you upgrade, try again, and see if it works for you?  I hadn't encountered this particular issue because the settings file in my projects is typically installed into site-packages before I run the dev server, so the relative path didn't matter.

**Comment by leehinde** [comment]
Yes, the update worked for me. Thanks.
